opentracing-util/src/main/java/io/opentracing/util/AutoFinishScopeManager.java

```diff
@@ -18,6 +18,13 @@
 
 import java.util.concurrent.atomic.AtomicInteger;
 
+/**
+ * @deprecated use {@link ThreadLocalScopeManager} instead.
```

Comment:
Does this explain you you can still 'automatically' finish spans when scope is closed?
Or do we not want that anymore?

i.e. Please describe the _reason_ for deprecating this functionality

Reply:
Added a (very) small note. Let me know if you think we need further clarification/details ;)